component.json

```diff
@@ -19,7 +19,8 @@
     "lang/no-nb.js",
     "lang/pl-PL.js",
     "lang/pt-PT.js",
-    "lang/sv-SE.js"
+    "lang/sv-SE.js",
+    "lang/zh-CN.js"
```

Comment:
Don't forget to add "lang/fr.js" and change from "lang/no-nb.js" to "lang/nb.js".